src/server/qgsconfigparserutils.cpp

```diff
@@ -77,14 +80,21 @@ void QgsConfigParserUtils::appendCrsElementToLayer( QDomElement& layerElement, c
   layerElement.insertAfter( crsElement, precedingElement );
 }
 
-void QgsConfigParserUtils::appendLayerBoundingBoxes( QDomElement& layerElem, QDomDocument& doc, const QgsRectangle& layerExtent,
+void QgsConfigParserUtils::appendLayerBoundingBoxes( QDomElement& layerElem, QDomDocument& doc, const QgsRectangle& lExtent,
```

Comment:
I miss the line `QgsRectangle layerExtent = lExtent;` than I suspect an unwanted change :-)